coc_collector.py

```diff
@@ -0,0 +1,101 @@
+import json
+import pandas as pd
+import requests
+from sqlalchemy import *
+from enum import Enum
+from bossdi_db import *
+from sqlalchemy.orm import sessionmaker
+from github import Github
+
+
+import json
+
+with open('ossdi.config.json') as config_file:
+    confidg_data = json.load(config_file)
+
+class OSSDI_Collector:
+    """
+    a class for getting metrics from the GitHub API
+    """
+    def __init__(self, api_key):
+        """
+        Creates a new GitHub instance
```

Comment:
i think you mean "github connection" cause instance would be a whole new website